### PR TITLE
upgrade: docker-compose to v2

### LINKED ISSUE #20: consider detecting / supporting docker compose (with space)
Hint:
https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose
Thanks

### PR DESCRIPTION
**Pull request by 9bany**
First, thank you for your work @isbang . This is an amazing tool.

I think we need to upgrade the docker-compose version to v2 because v1 is incompatible.
- docker-compose(v1: the current version use by this repo): no longer maintain, wrote by python [docker/compose/tree/v1.24.0](https://github.com/docker/compose/tree/v1.24.0/compose)
-  The migration of Docker was done with Docker Compose. Use the official plugin instead. [docs.docker.com/compose](https://docs.docker.com/compose/migrate/)

resolve #20

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   "dependencies": {
     "@actions/core": "^1.8.2",
     "@actions/github": "^4.0.0",
-    "docker-compose": "^0.23.5"
+    "docker-compose": "^0.24.0",
+    "yarn": "^1.22.19"
```
Comment (isbang) [inline package.json:19]: I hope to keep using npm. May I ask you to remove yarn dependency?

**Review by isbang — COMMENTED** [review]
Thank you for your contribution. I was barely able to maintaining this project, but now I think I should to focus on the transition to v2.

**Comment by isbang** [comment]
Please run `npm run format` before pushing your commit ;)

File: package.json (hunk 1 above)
Comment (9bany) [reply package.json]: just done

**Comment by 9bany** [comment]
thanks for your feedback @isbang 
Just done, please review again

**Review by isbang — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**